docs/taskbackground.rst

```diff
@@ -82,30 +215,41 @@ For each permutation, we calculate 20 features:
 #. ``qgl`` of the :math:`b_{top-had}` jet
 #. ``qgl`` of the :math:`b_{top-lep}` jet
 
-For each permutation, all 20 features are fed into a boosted decision tree, which was trained to select correct permutations. After this, the permutation with the highest BDT score is selected as "correct", then we use those jet-parton assignments to calculate the observables of interest.
+For each permutation, all 20 features are fed into a boosted decision tree, which was trained to select correct permutations. 
+After this, the permutation with the highest BDT score is selected as "correct", then we use those jet-parton assignments to calculate the observables of interest.
 
 It is a future goal to move onto a more sophisticated architecture, as the BDT method is restrictive since it becomes computationally expensive for events with high jet multiplicity.
 
 BDT Performance
```

Comment:
Let's add to this section some information about which BDT this is concretely showing (the one provided in our repository), since the results will differ if somebody re-implements this from scratch.